Animations/CMakeLists.txt

```diff
@@ -0,0 +1,220 @@
+# Configure vendored libraries
+set(DEAR_IMGUI_USER_CONFIG "${CMAKE_CURRENT_LIST_DIR}/include/core/InternalImGuiConfig.h")
+set(DEAR_IMGUI_EXTRA_INCLUDE_DIRS
+        "${CMAKE_CURRENT_LIST_DIR}/vendor/cppUtils/single_include"
+        "${CMAKE_CURRENT_LIST_DIR}/include")
+
+# Add all vendored libraries
+add_subdirectory(vendor)
```

Comment:
This is nice, I wasn't sure if it was common practice to do this and was planning on refactoring the build a bit later on to do this as well :)

Reply:
Yeah, this is usually how you do it. There is also `include`, but that does some rather weird things. CMake is a magical box